File: mdp_python/broker.py
import sys
from lib import MajordomoBroker

def main():
    """create and start new broker"""
    verbose = '-v' in sys.argv
    broker = MajordomoBroker(verbose)
    broker.bind('tcp://*:5555')
    broker.mediate()

if __name__ == '__main__':
    main()

File: mdp_rust/examples/broker.rs
extern crate majordomo;
extern crate pretty_env_logger;
use majordomo::MajordomoBroker;

fn main() {
    pretty_env_logger::init_timed();

    let mut broker = MajordomoBroker::new("tcp://*:5555", 1000).expect("Couldn't bind the broker");
    broker.start().expect("Error during broker activity...");
}
